packages/jellyfish-api-core/src/index.ts

```diff
@@ -35,6 +35,25 @@ export * as icxorderbook from './category/icxorderbook'
 export * as misc from './category/misc'
 export * as loan from './category/loan'
 
+export { AccountHistory } from './category/account'
+export { BlockchainInfo } from './category/blockchain'
+export {
+  AuctionPagination,
+  CollateralTokenDetail,
+  GetLoanSchemeResult,
+  LoanSchemeResult,
+  LoanTokenResult,
+  VaultActive,
+  VaultLiquidation,
+  VaultLiquidationBatch,
+  VaultPagination,
+  VaultState
+} from './category/loan'
+export { EstimateMode } from './category/mining'
+export { PoolPairInfo } from './category/poolpair'
+export { TokenInfo, TokenResult } from './category/token'
+export { MasternodeInfo, MasternodeState, MasternodeTimeLock } from './category/masternode'
```

Comment:
I am thinking whether:

1. we should just export everything and see how well it can deconflict?
2. used named exports? e.g: 
   1. `export * as account from './category/account'`
   2. `import account from '...'`
   3.  `account.AccountHistory`


Since this is explicitly declared just for whale-api, might seem arbitrary and hard to maintain?

Reply:
We could try going for `option 1` and see if we can't deconflict.

From memory the conflicts are usually related to the same interface being declared in different files. So I think it should be possible to DRY it up and create shared interfaces. I didn't do this as part of the original whale migration PR because it will require a lot more unrelated refactoring of jellyfish core packages. And the current implementation of this is the path of least resistance to get whale migrated without having to refactor any code.

I would say its probably the right direction to head though so we can correct some past mistakes in terms of our export structures.

Reply:
Perhaps we can get this in with a `todo` ticket for refactoring export structures in jellyfish-core?

This way we may prevent bloating time frames for whale consolidation so we can continue whale work within jellyfish and shift the ongoing refactoring to the right?

Reply:
Did an initial pass at deconflicting the exports so we can do a `export *` approach. All of them except for 1 was `UTXO` duplication. The one I am unsure about is `MasterNodeInfo`, for now I added the info coming from the master node file itself as the source of truth.